lib/logstash/filters/kv.rb

```diff
@@ -60,6 +60,30 @@ class LogStash::Filters::KV < LogStash::Filters::Base
   #     }
   config :trimkey, :validate => :string
 
+
+
+  # Normalize values to lower case, upper case or capitals.
+  #
+  # For example, to capitalize all values:
+  # [source,ruby]
+  #     filter {
+  #       kv {
+  #         norm => "capitalize"
+  #       }
+  #     }
+  config :norm, :validate => ["lowercase", "uppercase", "capitalize"]
```

Comment:
`norm` feels like the wrong name for this setting for two reasons. First, `norm` is abbreviated, and I prefer not to abbreviate setting names. Second, that I'm not sure `normalize' is the right name for the behavior. 

Maybe something like `key_letters` and `value_letters` for the setting names? Or `change_key` and `change_value` ?

```
key_letters => "lowercase"
# or this
change_key => "uppercase"
```